src/foam/u2/view/ColorPicker.js

```diff
@@ -46,8 +46,13 @@ foam.CLASS({
           if ( foam.Color.COLOR_TO_NAME[v] ) return foam.Color.COLOR_TO_NAME[v];
           return value;
         },
-        function (value) {
+        function(value) {
           if ( typeof value !== 'string' ) return value;
+          
+          // Resolve CSS tokens for display in the color picker
+          if ( foam.String.isInstance(value) && value.startsWith('$') ) {
```

Comment:
returnTokenValue already does a startsWith check so you can just call the function every time

Reply:
done.